### DISCUSSION TITLE
Argus PT camera?

### DISCUSSION THREAD

**Discussion by JocWal** [thirtythreeforty/neolink#216] [Q&A]
Did anyone manage to make this work with an Argus PT camera?

**Comment by QuantumEntangledAndy** [comment]
Sorry for not getting back to you sooner. Been busy with work. We have Argus Cameras working on this PR #199, but it is not yet merged into master.

**Reply by JocWal** [reply]
Hi, thanks a lot for your reply. I used your docker image to test this. However, when I use a uid entry with the corresponing camera uid as shown in the sample config, the container crashes telling me that it misses the "address" entry in the camera definition section. If I change 'uid' to address', the container can't connect to the camera either as it fails "to translate the camera address". The container is using the host network btw. 
Thanks for your help!!

**Reply by QuantumEntangledAndy** [reply]
Which docker image did you use? Please tell me the name as there are a few different ones and I want to make sure that it is the right one.

**Reply by QuantumEntangledAndy** [reply]
Ideally it should be this one:

```bash
docker pull quantumentangledandy/neolink:udp_docker
```

**Reply by JocWal** [reply]
Ah, I pulled the latest instead of the upd_docker one. I was able to connect now. Thanks a lot!

**Reply by QuantumEntangledAndy** [reply]
Oh good I hope it goes well :)
